docs/learn-staking.md

```diff
@@ -123,6 +123,15 @@ Based on the above rewards distribution, nominators in validator pool B get more
 than those in pool A because pool A has more overall stake. Sam has staked 50 DOT in pool A, but he
 only gets 8.3 in return, whereas Kitty gets 12.5 with the same amount of stake.
 
+There is an additional factor to consider in terms of rewards. While there is no limit to the number
+of nominators a validator may have, a validator does have a limit to how many nominators to which it
+can pay rewards. In Polkadot and Kusama, this limit is currently 64, although this can be modified
```

Comment:
Is the limit going to change anytime soon?

Reply:
Probably.  There's a PR to update it to 128 on Kusama and 256 on Polkadot.  But I don't want to mention this yet.